maintenance/key_value_pairs.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# -----------------------------------------------------------------------------
+#   Copyright (C) 2017 University of Dundee. All rights reserved.
+#
+#   This program is free software; you can redistribute it and/or modify
+#   it under the terms of the GNU General Public License as published by
+#   the Free Software Foundation; either version 2 of the License, or
+#   (at your option) any later version.
+#   This program is distributed in the hope that it will be useful,
+#   but WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#   GNU General Public License for more details.
+#
+#   You should have received a copy of the GNU General Public License along
+#   with this program; if not, write to the Free Software Foundation, Inc.,
+#   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# ------------------------------------------------------------------------------
+
+"""
+This script adds Key-Value Pairs to images in a Dataset with the specified name 
+for 40 users (user-1 through user-40). The Key-Value Pairs are defined in the
+variables kvp_setx and are added to the images inside the dataset according to
+the features in the images in the list images_kvp_order.
+The script could be simplified by adding the Key-Value Pairs randomly to the
+images in the Dataset.
+"""
+import argparse
+
```

Comment:
Why is an empty line here ? the ``cleanup.py`` has no empty line in this place for example